<launch>
  <arg name="robot_0" default="tb3_0" />
  <arg name="robot_1" default="tb3_1" />
  <arg name="robot_2" default="tb3_2" />

  <include file="$(find ff_examples_ros1)/launch/include/move_base_namespace.launch">
  <arg name="robot_namespace" value="$(arg robot_0)" />
  </include>

  <include file="$(find ff_examples_ros1)/launch/include/move_base_namespace.launch">
  <arg name="robot_namespace" value="$(arg robot_1)" />
  </include>

  <include file="$(find ff_examples_ros1)/launch/include/move_base_namespace.launch">
  <arg name="robot_namespace" value="$(arg robot_2)" />
  </include>
</launch>	